### ISSUE TITLE
Add audio balance adjustment

### ISSUE THREAD

**Issue by andiandi13** [vanilla-music/vanilla#967]
Adjusting audio balance can be useful both for people who have some earing loss on one ear, as well as when one side of the headset is quieter than the other.

I don't know if it's possible to implement such an option at all, but it'd be very helpful if it is.

**Comment by mindforger** [comment]
slap me if i am wrong, but isn't an equalizer part of the phone specific OS? What phone do you have?

**Comment by andiandi13** [comment]
Yes thats right, and I heard that in some devices there is an audio balance adjustment feature.

I have a Nokia 8 on Nougat and an Samsung Galaxy S5. There isn't such option

**Comment by mindforger** [comment]
https://www.reddit.com/r/Nokia/comments/8yrq7d/does_the_nokia_8_have_an_equalizer_if_it_does/

this should help you directly to look for a workarounf, but to my understanding it would be nice if vanilla would provide that API function to open the system equalizer if the API is available

**Comment by andiandi13** [comment]
It's not what I meant. I have an equalizer but there is not the audio balance option.

**Comment by andiandi13** [comment]
In some phones it is part, but not in all of them. So it'd would be nice if it can be in Vanilla.

**Comment by adrian-bl** [comment]
Thats a good idea: should be doable as we already have an aufiofx session open for gapless: changing the volume (as in: reducing) on one channel should be doable
